telegram/constants.py

```diff
@@ -486,24 +525,103 @@ class InlineQueryResultType(StringEnum):
 
 class LocationLimit(IntEnum):
     """This enum contains limitations for :class:`telegram.Location`/
-    :meth:`telegram.Bot.send_location`. The enum members of this enumeration are instances
-    of :class:`int` and can be treated as such.
+    :meth:`telegram.Bot.edit_message_live_location`/:meth:`telegram.Bot.send_location`.
+    The enum members of this enumeration are instances of :class:`int` and can be treated as such.
 
     .. versionadded:: 20.0
     """
 
     __slots__ = ()
 
     HORIZONTAL_ACCURACY = 1500
-    """:obj:`int`: Maximum radius of uncertainty for the location, measured in meters."""
+    """:obj:`int`: Maximum value allowed for:
+
+    * :paramref:`~telegram.Location.horizontal_accuracy` parameter of :class:`telegram.Location`
+    * :paramref:`~telegram.InlineQueryResultLocation.horizontal_accuracy` parameter of
```

Comment:
do these bulleted lists also render correctly? somethimes restructered text is a bit picky when it comes to this - at least in my experience …

Reply:
They do render (at least locally), yes

Reply:
![Screenshot from 2022-11-09 20-15-01](https://user-images.githubusercontent.com/58207913/200899323-6e220d4e-86e3-4f1d-bff2-b41dd5b8a6b8.png)